cmd/flags.go

```diff
@@ -42,7 +42,6 @@ var GrafanaFlags = []cli.Flag{
 		Name:     "grafana",
 		Usage:    "If set, initialize Grafana",
 		Required: false,
-		Value:    true,
```

Comment:
I think I still want to implicitely build grafana by default and require an explicit disabling of that flag. WDYT?

Reply:
In this case won't it always build both enterprise and oss when only --enterprise is used? We would need to have something like --enterprise --no-oss? I thought it would be better to have it as `false` consider the default action to build, and that flag would work similar to something like `--force-oss` (so it always builds oss even on cases that it normally shouldn't)

Reply:
I would want `--enterprise` to build both, but `--enterprise --grafana=false` to only build enterprise.

More importantly, I want

`go run ./cmd package --distro=linux/amd64` to give me a Grafana package.

Reply:
Oh! Ok we can set --grafana=false! That makes sense!
But isn't it weird that the arg is "grafana" and not "oss", you're getting Grafana either way.

Reply:
Would it perhaps make sense to do a bit of renaming?

- `--distro` -> `--platform`
- `--distribution` as list allowing "oss" and "enterprise"?

Reply:
I readded the default value to the grafana flag.

Reply:
I feel like `--oss` is less descriptive but "--grafana" is also a little ambiguous. It would make more sense if it was '--grafana-enterprise' instead of '--enterprise' but I felt like that was too redundant to have '--grafana' and '--grafana-enterprise', so it was just shorted to '--enterprise' :joy:.

Open to suggestions though maybe we can discuss this in another issue or in Slack.

@zerok I like sticking with `distro` because that's consistent with what Go calls it (`go tool dist list` gives a bunch of valid values for the `distro` flag). Although with that logic it should probably be called "dist" instead of "distro".

I am open to the idea of adding a string flag to distinguish grafana/enterprise/other variations. We don't have a standard term that describes the relationship. Some options are `edition` and `flavor`.